pyciemss/integration_utils/result_processing.py

```diff
@@ -29,12 +49,12 @@ def convert_to_output_format(samples: Dict[str, torch.Tensor]) -> pd.DataFrame:
         if sample.ndim == 1:
             # Any 1D array is a sample from the distribution over parameters.
             # Any 2D array is a sample from the distribution over states, unless it's a model weight.
-            name = name + "_param"
+            name = name + "_param" if not name.endswith("_param") else name
             pyciemss_results["parameters"][name] = (
                 sample.data.detach().cpu().numpy().astype(np.float64)
             )
         else:
-            name = name + "_state"
+            name = name + "_state" if not (name.endswith("_state")) else name
```

Comment:
Nice! I was going to look into why `_state` was being repeated twice, but this does it.

Reply:
A whole bunch fiddly little things in this world. 

I had another one that didn't append `_state` if it was an `_observable` but that wasn't obviously the right thing to do (it made labels look a little better which is why I did it) AND broke other tests so I changed that back.  Something to consider in the future.